care/emr/resources/user/spec.py

```
@@ -111,8 +112,25 @@ def perform_extra_serialization(cls, mapping, obj: User):
         from care.emr.resources.organization.spec import OrganizationReadSpec
 
         super().perform_extra_serialization(mapping, obj)
-        if obj.created_by:
-            mapping["created_by"] = UserSpec.serialize(obj.created_by).to_json()
+
+        CACHED_CREATED_BY = f"user:{obj.external_id}:data"
```

Ruff (0.8.2)

Variable `CACHED_CREATED_BY` in function should be lowercase

(N806)